pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -8122,6 +8123,44 @@
   accessibility_card_title_10: "Quorum "
   accessibility_card_text_10: "An adaptation of Code.org’s AP Computer Science Principles curriculum using Quorum, an evidence-based language originally designed to provide accessibility for blind or visually-impaired students."
 
+  engineering_hero_heading: "Join our engineering team!"
+  engineering_hero_desc: "As we continue to expand our engineering team, we wanted to share more about our engineering organization at Code.org and some of the unique aspects of working here. If you are interested in applying for one of our roles, we'd love to hear from you!"
+  engineering_team_title: "Meet the engineering team at Code.org"
+  engineering_team_card_text_1: "This is (most of) us at our in-person get together during the summer of 2023! This crowd is divided into four sub-teams within our engineering team who work together to build and serve the platform for all our students and teachers.This is (most of) us at our in-person get together during the summer of 2023! This crowd is divided into four sub-teams within our engineering team who work together to build and serve the platform for all our students and teachers."
```

Comment:
This paragraph is duplicated `This is (most of) us at our in-person get together during the summer of 2023! This crowd is divided into four sub-teams within our engineering team who work together to build and serve the platform for all our students and teachers.`